script.js

```diff
@@ -1 +1,152 @@
+//yehey 하이
+const todoForm = document.querySelector(".submitContainer"),
+    todoInput = todoForm.querySelector("input"),
+    todoList = document.querySelector(".todoListContainer"),
+    completeList = document.querySelector(".completeContainer");
 
+const TODOS_LS = "todos";
+let todos =[];    //여러개의 할 일들을 담을 배열
+const COM_LS = 'done';
+let done=[];
+
+function loadToDos(){
+    const localTodo = localStorage.getItem(TODOS_LS);
+    const localDone = localStorage.getItem(COM_LS);
+    if(localTodo!==null){
+        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
+        const local = JSON.parse(localTodo);
+        local.forEach(function(toDo){ addToDo(toDo.text);})
+    }
+    if(localDone!==null){
+        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
+        const local = JSON.parse(localDone);
+        local.forEach(function(toDo){ addComplete(toDo.text);})
+    }
+}
+//text 기반으로 todolist에 할 일 추가
+function addToDo(text){     
+    const todo = document.createElement("li");
+    const delBtn = document.createElement("button");
+    //delBtn.innerText = "";
+    delBtn.classList.add("deleteimg");
+    const completeBtn = document.createElement("button");
+    completeBtn.classList.add("completeimg");
+    const span = document.createElement("span");
+    span.innerText = text;
+    const todoId = todos.length+1;
+
+    delBtn.addEventListener("click", deleteTodo);
+    completeBtn.addEventListener("click", completeTodo);
+
+    todo.appendChild(span);
+    todo.appendChild(delBtn);
+    todo.appendChild(completeBtn);
+    todo.id=todoId;
+    todo.classList.add("todoList");
+
+    todoList.appendChild(todo);
+
+    const todoObj = {
+        text : text,
+        id: todoId,
+    };
+
+    todos.push(todoObj);
+    saveToLocal('todo');
+}
+
+//todo 리스트에서 할일 끝내면 complete 리스트에 할일을 text 기반으로 추가 
+function addComplete(text){
```

Comment:
```suggestion
function moveToCompletedList(item) {
```

함수 이름은 늘 일관성있고, 구체적으로, 자세하게 적어주시면 됩니다~!